docs/update_release_notes.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Script to help with the SMQTK release process. Performs the following steps:
+#   - Poetry version (major, minor, or patch)
+#   - Rename release_notes/pending_release file to release_notes/version
+#   - Add reference to new release notes file in release_notes.rst
+#   - Add new release notes stub file
+#
+# Two git commits are created. One for the version bump and one for the new
+# release notes stub file.
+#
+
```

Comment:
Around here, define a `SCRIPT_DIR` variable as follows. This lets us be a little more robust to invocation location than using explicit relative paths. Otherwise we have to run this script with the CWD being the `docs/` directory or it won't work / do the wrong thing.
```bash
SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

When using paths to files below, we should reference them in relation to this path, e.g. on L26 -> `RELEASE_NOTES_PATH="${SCRIPT_DIR}/release_notes"`